surrealdb/clients/ws.py

```diff
@@ -33,10 +33,10 @@
 from ..models import RPCRequest
 from ..models import RPCResponse
 
-__all__ = ("SurrealDBWSClient",)
+__all__ = ("SurrealDBWebsocketClient",)
 
 
-class SurrealDBWSClient:
+class SurrealDBWebsocketClient:
```

Comment:
Why does it need to be preceeded by `SurrealDB` ?

Reply:
see https://github.com/surrealdb/surrealdb.py/pull/6#discussion_r973738089